demo/src/index.js

```diff
@@ -1,14 +1,12 @@
-
 import mirador from 'mirador/dist/es/src/index';
 import annotationPlugins from '../../src';
 import LocalStorageAdapter from '../../src/LocalStorageAdapter';
-import AnnototAdapter from '../../src/AnnototAdapter';
+import RerumAdapter from '../../src/RerumAdapter';
 
-const endpointUrl = 'http://127.0.0.1:3000/annotations';
 const config = {
   annotation: {
     adapter: (canvasId) => new LocalStorageAdapter(`localStorage://?canvasId=${canvasId}`),
-    // adapter: (canvasId) => new AnnototAdapter(canvasId, endpointUrl),
+    // adapter: (canvasId) => new RerumAdapter(canvasId), // put Annotations online with RERUM
```

Comment:
recommend including optional params in the comment for completeness:

```
// adapter: (canvasId) => new RerumAdapter(canvasId, endpointUri?, creator?), // put Annotations online with RERUM
```

Reply:
argh dang, I removed them all when I had it uncommented for the functionality.  That's a fine idea to put the params into the comment, that won't hurt anything.